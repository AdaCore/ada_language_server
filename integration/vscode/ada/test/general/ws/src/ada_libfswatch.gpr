--  This project has been generated by GPRINSTALL Pro 26.0w (20241106) (x86_64-pc-linux-gnu)

with "c_libfswatch";
with "c_libfswatch";
with "gnatcoll_core";
with "gnatcoll_minimal";

standard project Ada_Libfswatch is
   type BUILD_KIND is ("default");
   BUILD : BUILD_KIND := external("ADA_LIBFSWATCH_BUILD", "default");

   for Languages use ("Ada");

   case BUILD is
      when "default" =>
         for Source_Dirs use ("../../include/ada_libfswatch");
         for Object_Dir use "../../lib/ada_libfswatch";
   end case;

   package Naming is
      case BUILD is
         when "default" =>
            null;
      end case;
   end Naming;

   package Linker is
      case BUILD is
         when "default" =>
            null;
      end case;
   end Linker;

   package Install is
      for Active use "False";
   end Install;

   for Externally_Built use "True";

end Ada_Libfswatch;
