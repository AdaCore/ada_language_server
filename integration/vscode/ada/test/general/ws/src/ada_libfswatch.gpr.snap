>--  This project has been generated by GPRINSTALL Pro 26.0w (20241106) (x86_64-pc-linux-gnu)
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.gpr comment.line.double-dash.gpr
>
>with "c_libfswatch";
#^^^^ source.gpr meta.clause.with.gpr keyword.gpr
#    ^ source.gpr meta.clause.with.gpr
#     ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#      ^^^^^^^^^^^^ source.gpr meta.clause.with.gpr string.quoted.double.gpr
#                  ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                   ^ source.gpr meta.clause.with.gpr punctuation.gpr
>with "c_libfswatch";
#^^^^ source.gpr meta.clause.with.gpr keyword.gpr
#    ^ source.gpr meta.clause.with.gpr
#     ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#      ^^^^^^^^^^^^ source.gpr meta.clause.with.gpr string.quoted.double.gpr
#                  ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                   ^ source.gpr meta.clause.with.gpr punctuation.gpr
>with "gnatcoll_core";
#^^^^ source.gpr meta.clause.with.gpr keyword.gpr
#    ^ source.gpr meta.clause.with.gpr
#     ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#      ^^^^^^^^^^^^^ source.gpr meta.clause.with.gpr string.quoted.double.gpr
#                   ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                    ^ source.gpr meta.clause.with.gpr punctuation.gpr
>with "gnatcoll_minimal";
#^^^^ source.gpr meta.clause.with.gpr keyword.gpr
#    ^ source.gpr meta.clause.with.gpr
#     ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#      ^^^^^^^^^^^^^^^^ source.gpr meta.clause.with.gpr string.quoted.double.gpr
#                      ^ source.gpr meta.clause.with.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                       ^ source.gpr meta.clause.with.gpr punctuation.gpr
>
>standard project Ada_Libfswatch is
#^^^^^^^^ source.gpr meta.project.gpr keyword.gpr
#        ^ source.gpr meta.project.gpr
#         ^^^^^^^ source.gpr meta.project.gpr keyword.gpr
#                ^ source.gpr meta.project.gpr
#                 ^^^^^^^^^^^^^^ source.gpr meta.project.gpr entity.name.project.gpr entity.name.gpr
#                               ^ source.gpr meta.project.gpr
#                                ^^ source.gpr meta.project.gpr keyword.gpr
>   type BUILD_KIND is ("default");
#^^^ source.gpr meta.project.gpr
#   ^^^^ source.gpr meta.project.gpr meta.declaration.type.gpr keyword.gpr
#       ^ source.gpr meta.project.gpr meta.declaration.type.gpr
#        ^^^^^^^^^^ source.gpr meta.project.gpr meta.declaration.type.gpr entity.name.type.gpr entity.name.gpr
#                  ^ source.gpr meta.project.gpr meta.declaration.type.gpr
#                   ^^ source.gpr meta.project.gpr meta.declaration.type.gpr keyword.gpr
#                     ^ source.gpr meta.project.gpr meta.declaration.type.gpr
#                      ^ source.gpr meta.project.gpr meta.declaration.type.gpr punctuation.gpr
#                       ^ source.gpr meta.project.gpr meta.declaration.type.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                        ^^^^^^^ source.gpr meta.project.gpr meta.declaration.type.gpr string.quoted.double.gpr
#                               ^ source.gpr meta.project.gpr meta.declaration.type.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                ^ source.gpr meta.project.gpr meta.declaration.type.gpr punctuation.gpr
#                                 ^ source.gpr meta.project.gpr meta.declaration.type.gpr punctuation.gpr
>   BUILD : BUILD_KIND := external("ADA_LIBFSWATCH_BUILD", "default");
#^^^ source.gpr meta.project.gpr
#   ^^^^^ source.gpr meta.project.gpr meta.declaration.variable.gpr variable.name.gpr entity.name.gpr
#        ^ source.gpr meta.project.gpr meta.declaration.variable.gpr
#         ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.declaration.variable.type.gpr punctuation.gpr
#          ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.declaration.variable.type.gpr
#           ^^^^^^^^^^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.declaration.variable.type.gpr entity.name.type.gpr entity.name.gpr
#                     ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.declaration.variable.type.gpr
#                      ^^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr punctuation.gpr
#                        ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr
#                         ^^^^^^^^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr variable.language.gpr
#                                 ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr punctuation.gpr
#                                  ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                   ^^^^^^^^^^^^^^^^^^^^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr
#                                                       ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                                        ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr punctuation.gpr
#                                                         ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr
#                                                          ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                                           ^^^^^^^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr
#                                                                  ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                                                   ^ source.gpr meta.project.gpr meta.declaration.variable.gpr meta.assignment.gpr meta.external.gpr meta.string_list.gpr punctuation.gpr
#                                                                    ^ source.gpr meta.project.gpr meta.declaration.variable.gpr punctuation.gpr
>
>   for Languages use ("Ada");
#^^^ source.gpr meta.project.gpr
#   ^^^ source.gpr meta.project.gpr meta.attribute.gpr keyword.gpr
#      ^ source.gpr meta.project.gpr meta.attribute.gpr
#       ^^^^^^^^^ source.gpr meta.project.gpr meta.attribute.gpr entity.other.attribute-name.gpr
#                ^ source.gpr meta.project.gpr meta.attribute.gpr
#                 ^^^ source.gpr meta.project.gpr meta.attribute.gpr keyword.gpr
#                    ^ source.gpr meta.project.gpr meta.attribute.gpr
#                     ^ source.gpr meta.project.gpr meta.attribute.gpr meta.string_list.gpr punctuation.gpr
#                      ^ source.gpr meta.project.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                       ^^^ source.gpr meta.project.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr
#                          ^ source.gpr meta.project.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                           ^ source.gpr meta.project.gpr meta.attribute.gpr meta.string_list.gpr punctuation.gpr
#                            ^ source.gpr meta.project.gpr meta.attribute.gpr punctuation.gpr
>
>   case BUILD is
#^^^ source.gpr meta.project.gpr
#   ^^^^ source.gpr meta.project.gpr meta.case.gpr keyword.control.gpr
#       ^ source.gpr meta.project.gpr meta.case.gpr
#        ^^^^^ source.gpr meta.project.gpr meta.case.gpr variable.name.gpr variable.name.gpr
#             ^ source.gpr meta.project.gpr meta.case.gpr
#              ^^ source.gpr meta.project.gpr meta.case.gpr keyword.control.gpr
>      when "default" =>
#^^^^^^ source.gpr meta.project.gpr meta.case.gpr
#      ^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
#          ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr
#           ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#            ^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr
#                   ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                    ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr
#                     ^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
>         for Source_Dirs use ("../../include/ada_libfswatch");
#^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr
#         ^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr keyword.gpr
#            ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#             ^^^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr entity.other.attribute-name.gpr
#                        ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#                         ^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr keyword.gpr
#                            ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#                             ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr meta.string_list.gpr punctuation.gpr
#                              ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr
#                                                           ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr meta.string_list.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                                            ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr meta.string_list.gpr punctuation.gpr
#                                                             ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr punctuation.gpr
>         for Object_Dir use "../../lib/ada_libfswatch";
#^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr
#         ^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr keyword.gpr
#            ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#             ^^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr entity.other.attribute-name.gpr
#                       ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#                        ^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr keyword.gpr
#                           ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr
#                            ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                             ^^^^^^^^^^^^^^^^^^^^^^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr string.quoted.double.gpr
#                                                     ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                                      ^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr meta.attribute.gpr punctuation.gpr
>   end case;
#^^^ source.gpr meta.project.gpr meta.case.gpr meta.case.when.gpr
#   ^^^ source.gpr meta.project.gpr meta.case.gpr keyword.control.gpr
#      ^ source.gpr meta.project.gpr meta.case.gpr
#       ^^^^ source.gpr meta.project.gpr meta.case.gpr keyword.control.gpr
#           ^ source.gpr meta.project.gpr meta.case.gpr punctuation.gpr
>
>   package Naming is
#^^^ source.gpr meta.project.gpr
#   ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#          ^ source.gpr meta.project.gpr meta.package.gpr
#           ^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#                 ^ source.gpr meta.project.gpr meta.package.gpr
#                  ^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
>      case BUILD is
#^^^^^^ source.gpr meta.project.gpr meta.package.gpr
#      ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#          ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#           ^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr variable.name.gpr variable.name.gpr
#                ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#                 ^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
>         when "default" =>
#^^^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#         ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
#             ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#              ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#               ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr
#                      ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                       ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#                        ^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
>            null;
#^^^^^^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#            ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.gpr
#                ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
>      end case;
#^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#      ^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#         ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#          ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#              ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr punctuation.gpr
>   end Naming;
#^^^ source.gpr meta.project.gpr meta.package.gpr
#   ^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#      ^ source.gpr meta.project.gpr meta.package.gpr
#       ^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#             ^ source.gpr meta.project.gpr meta.package.gpr punctuation.gpr
>
>   package Linker is
#^^^ source.gpr meta.project.gpr
#   ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#          ^ source.gpr meta.project.gpr meta.package.gpr
#           ^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#                 ^ source.gpr meta.project.gpr meta.package.gpr
#                  ^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
>      case BUILD is
#^^^^^^ source.gpr meta.project.gpr meta.package.gpr
#      ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#          ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#           ^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr variable.name.gpr variable.name.gpr
#                ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#                 ^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
>         when "default" =>
#^^^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#         ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
#             ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#              ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#               ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr
#                      ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                       ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#                        ^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.control.gpr
>            null;
#^^^^^^^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#            ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr keyword.gpr
#                ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
>      end case;
#^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr meta.case.when.gpr
#      ^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#         ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr
#          ^^^^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr keyword.control.gpr
#              ^ source.gpr meta.project.gpr meta.package.gpr meta.case.gpr punctuation.gpr
>   end Linker;
#^^^ source.gpr meta.project.gpr meta.package.gpr
#   ^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#      ^ source.gpr meta.project.gpr meta.package.gpr
#       ^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#             ^ source.gpr meta.project.gpr meta.package.gpr punctuation.gpr
>
>   package Install is
#^^^ source.gpr meta.project.gpr
#   ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#          ^ source.gpr meta.project.gpr meta.package.gpr
#           ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#                  ^ source.gpr meta.project.gpr meta.package.gpr
#                   ^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
>      for Active use "False";
#^^^^^^ source.gpr meta.project.gpr meta.package.gpr
#      ^^^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr keyword.gpr
#         ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr
#          ^^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr entity.other.attribute-name.gpr
#                ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr
#                 ^^^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr keyword.gpr
#                    ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr
#                     ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                      ^^^^^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr string.quoted.double.gpr
#                           ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                            ^ source.gpr meta.project.gpr meta.package.gpr meta.attribute.gpr punctuation.gpr
>   end Install;
#^^^ source.gpr meta.project.gpr meta.package.gpr
#   ^^^ source.gpr meta.project.gpr meta.package.gpr keyword.gpr
#      ^ source.gpr meta.project.gpr meta.package.gpr
#       ^^^^^^^ source.gpr meta.project.gpr meta.package.gpr entity.name.package.gpr
#              ^ source.gpr meta.project.gpr meta.package.gpr punctuation.gpr
>
>   for Externally_Built use "True";
#^^^ source.gpr meta.project.gpr
#   ^^^ source.gpr meta.project.gpr meta.attribute.gpr keyword.gpr
#      ^ source.gpr meta.project.gpr meta.attribute.gpr
#       ^^^^^^^^^^^^^^^^ source.gpr meta.project.gpr meta.attribute.gpr entity.other.attribute-name.gpr
#                       ^ source.gpr meta.project.gpr meta.attribute.gpr
#                        ^^^ source.gpr meta.project.gpr meta.attribute.gpr keyword.gpr
#                           ^ source.gpr meta.project.gpr meta.attribute.gpr
#                            ^ source.gpr meta.project.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                             ^^^^ source.gpr meta.project.gpr meta.attribute.gpr string.quoted.double.gpr
#                                 ^ source.gpr meta.project.gpr meta.attribute.gpr string.quoted.double.gpr punctuation.definition.string.gpr
#                                  ^ source.gpr meta.project.gpr meta.attribute.gpr punctuation.gpr
>
>end Ada_Libfswatch;
#^^^ source.gpr meta.project.gpr keyword.gpr
#   ^ source.gpr meta.project.gpr
#    ^^^^^^^^^^^^^^ source.gpr meta.project.gpr entity.name.project.gpr
#                  ^ source.gpr meta.project.gpr punctuation.gpr
>